project/profiles/capacity4more/modules/c4m/content/c4m_content_book/templates/c4m-content-book-node-export-html.tpl.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Template for the book export.
+ */
+?>
+<div class="book-print--item">
+  <?php if (!empty($hierarchy)): ?>
+      <div class="book-print--navigation"><?php print $hierarchy; ?></div>
+  <?php endif; ?>
+  <?php print render($content); ?>
+</div>
+<?php if (!empty($children)) :?>
+    <div class="book-print--children"><?php print render($children); ?></div>
+<?php endif; ?>
```

Comment:
- Expected 1 newline at end of file; 2 found
- Additional whitespace found at end of file